docs/developers/documentation/index.md

```diff
@@ -92,6 +92,14 @@ Since the API reference documentation (autogenerated from the napari code docstr
 - a development installation of napari. Follow the [contributor guide](dev-installation) for details on how to do this;
 - the napari documentation requirements. Starting at the folder where you cloned the `napari/docs` repository, and after activating your clean `conda` environment, you can install these with `python -m pip install -r requirements.txt`.
 
+````{note}
+Due to the tool we use to build our [examples gallery](gallery)
+([sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html)), you may
+need to install napari from source without using an editable install if your
+goal is to build the documentation locally. In that case, you should not use the
+`-e` argument of `pip install`.
```

Comment:
Thanks for this @melissawm! I think without the context of #207 and the comments, this is a bit vague. I suggest adding language like: "If you see an error like the following: `AttributeError: module 'napari.Viewer' has no attribute 'close_all'`, then make sure you install napari *without* an editable install. That is, don't use`-e` to install a local version of napari. For more details, see the discussion at [napari/docs#207](https://github.com/napari/docs/pull/207#issuecomment-1648120055)."

Reply:
Yeah, I am debating not expanding too much to not be overwhelming for someone who only wants to build docs and giving full context. Maybe this isn't even necessary if we find out the source of the original error? I'll mark this PR as draft and explore a bit.